xml/ha_storage_protection.xml

```diff
@@ -42,422 +58,556 @@
+   </listitem>
+   <listitem>
+    <para> The shared storage segment <emphasis>must not</emphasis>
+     use host-based RAID, LVM2, nor DRBD*. DRBD can be splitted, which is
+     problematic for SBD, as there cannot be two states in SBD.
+     Cluster multi-device (Cluster MD) cannot be used for SBD.
+    </para>
+   </listitem>
+   <listitem>
+    <para> However, using storage-based RAID and multipathing is
+     recommended for increased reliability. </para>
+   </listitem>
+   <listitem>
+    <para>An SBD device can be shared between different clusters, as
+     long as no more than 255 nodes share the device. </para>
+   </listitem>
+   <listitem>
+    <para>For clusters with more than two nodes, you can also use SBD in
+    <emphasis>diskless</emphasis> mode.<remark>taroth 2018-04-18: add link to
+    this (new) section after it has been added</remark>
    </para>
-   <procedure>
-    <step>
-       <para>
-         <xref linkend="pro.ha.storage.protect.watchdog" xrefstyle="select:title"/>
-       </para>
-    </step>
-    <step>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+ <sect1 xml:id="sec.ha.storage.protect.fencing.number">
+  <title>Number of SBD Devices</title>
+  <!--fate#309375-->
+  <para> SBD supports the use of up to three devices: </para>
+  <variablelist>
+   <varlistentry>
+    <term>Diskless</term>
+    <listitem>
+     <para>&sbd-diskless;</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>One Device</term>
+    <listitem>
      <para>
-      <xref linkend="pro.ha.storage.protect.sbd.create" xrefstyle="select:title"/>
+      The most simple implementation. It is appropriate for clusters where
+      all of your data is on the same shared storage.
      </para>
-    </step>
-    <step>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Two Devices</term>
+    <listitem>
      <para>
-      <xref linkend="pro.ha.storage.protect.sbd.daemon" xrefstyle="select:title"/>
+      This configuration is primarily useful for environments that use
+      host-based mirroring but where no third storage device is available.
+      SBD will not terminate itself if it loses access to one mirror leg,
+      allowing the cluster to continue. However, since SBD does not have
+      enough knowledge to detect an asymmetric split of the storage, it
+      will not fence the other side while only one mirror leg is available.
+      Thus, it cannot automatically tolerate a second failure while one of
+      the storage arrays is down.
      </para>
-    </step>
-    <step>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Three Devices</term>
+    <listitem>
      <para>
-      <xref linkend="pro.ha.storage.protect.sbd.test" xrefstyle="select:title"/>
+      The most reliable configuration. It is resilient against outages of
+      one device&mdash;be it because of failures or maintenance. SBD
+      will only terminate itself if more than one device is lost. If at least
```

Comment:
SBD will possibly terminate itself only if more than one device are lost.

(-- Since "Pacemaker integration", which is also described in above, is  enabled by default, even if more than one SBD device are lost, SBD terminates itself only if necessary depending on the status of the cluster partition/node).

Reply:
Fixed, thanks.